// The purpose of this file is to check code running before any runtime errors in my try catch
// should output 2 2

void main (){
    try {
        int i = 0;
        while(i < 2){
            i = i + 1;
        }
        string s = itos(i);
        print(s);
        print("\n");
        print(stoi(s));
    }
    catch {
        print("ERROR");
    }
}